src/main/java/com/kobylynskyi/graphql/codegen/model/MultiLanguageDeprecated.java

```diff
@@ -0,0 +1,31 @@
+package com.kobylynskyi.graphql.codegen.model;
+
+import graphql.language.Directive;
+import graphql.language.StringValue;
+
+/**
+ * @author liguobin@growingio.com
+ * @version 1.0, 2020/12/28
+ */
+public class MultiLanguageDeprecated {
+
+    public static DeprecatedDefinition getLanguageDeprecated(GeneratedLanguage generatedLanguage, Directive directive) {
+        String JAVA = "Deprecated";
+        String SCALA = "deprecated";
+        String KOTLIN = "Deprecated";
+        String msg = null;
+        if (directive.getArguments().parallelStream().anyMatch(argument -> argument.getName().equals("reason"))) {
```

Comment:
1. Please remove `.parallelStream()` as it negatively impacts performance in our case.
2. Extract "reason" as constant: `private static final`

Reply:
I don't get it? Is it ```msg```?

Reply:
No, msg is fine. I am taking about parallel streams. They should be removed.

Reply:
Is it because of the stream order? I've taken it all out

Reply:
Thanks.
FYI: https://stackoverflow.com/questions/20375176/should-i-always-use-a-parallel-stream-when-possible